bodhi/server/util.py

```diff
@@ -897,7 +897,6 @@ def _end_session(self):
         This has been split off the main __call__ method to make it easier to
         mock it out in unit tests.
         """
-        self.session.close()
         Session.remove()
```

Comment:
By default Session.remove() calls Session.close() so we don't need to call it twice here.